ktor-io/common/src/io/ktor/utils/io/core/Buffers.kt

```diff
@@ -17,6 +17,8 @@ public expect class IoBuffer : Input, Output, ChunkBuffer {
     @Suppress("ConvertSecondaryConstructorToPrimary")
     public constructor(memory: Memory, origin: ChunkBuffer?)
```

Comment:
Please deprecate old constructor so that we can remove it in future releases

Reply:
We have no public alternative available, it's possible to do only in 1.6.0